### DISCUSSION TITLE
API 支持情况

### DISCUSSION THREAD

**Discussion by xyyyzzz** [NervJS/taro#16081] [Q & A] [answered]
![image](https://github.com/NervJS/taro/assets/45732128/a8197182-6e05-4262-8655-ebe94b27141f)
![image](https://github.com/NervJS/taro/assets/45732128/8c898afe-add7-47f1-8feb-834870418e90)

官网上显示的API支持情况，有颜色的图标代表支持、灰色代表不支持吗？还有为什么有些API页面显示的图标比别的页面多？

**Comment by honlyHuang** [comment, marked as answer]
颜色的图标代表支持、灰色代表不支持；为什么有的多有的少，多的是经过开发者或者使用者确认过的，在使用的过程中如果恰好使用了，但页面里缺的，可以帮忙补充的咯~
